app/lib/ui/widgets/utility/email_form.dart

```diff
@@ -98,25 +109,29 @@ class _EmailForm extends State<EmailForm> {
                       border: Border.all(color: DarkerBeige),
                       borderRadius: BorderRadius.all(Radius.circular(5.0))),
                   child: Text(
-                    'By clicking send a composed email will open in your default email app',
+                    'Note: Send button below will open Gmail',
```

Comment:
- [ ] Gmail may not be present on iPhones that use the iOS mailer, this message would be better as:
"Clicking send will open your device's email application"

Reply:
Done